README.md

```diff
@@ -71,14 +71,36 @@ be found in [Loadero wiki](https://wiki.loadero.com/docs/nightwatch/custom-comma
-| `takeScreenshot`          | Screenshot will be saved locally in project root directory                          |
-| `timeExecution`           | Execution time will be logged, but not saved                                        |
-| `updateNetwork`           | Network settings will not be updated                                                |
-| `waitForDownloadFinished` | Function will finish instantly and not wait for download to be finished             |
+| Command                   | Differences                                                                                                                                   |
+| ------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------- |
+| `genEmail`                | Full email or only first part can be provided, if only the first part is provided than `@mailinator` is appended                              |
+| `ignoreAlert`             | No differences                                                                                                                                |
+| `performTimed`            | No differences                                                                                                                                |
+| `receiveEmail`            | By default an empty array will be returned, but `emails.json` file can be saved in the working directory which will contain all of the emails |
+| `setFile`                 | Any local file can be used, migration to Loadero may require changing the file path                                                           |
+| `setRequestHeader`        | No request header will be set                                                                                                                 |
+| `setUserAgent`            | User agent won't be changed                                                                                                                   |
+| `takeScreenshot`          | Screenshot will be saved locally in project root directory                                                                                    |
+| `timeExecution`           | Execution time will be logged, but not saved                                                                                                  |
+| `updateNetwork`           | Network settings will not be updated                                                                                                          |
+| `waitForDownloadFinished` | Function will finish instantly and not wait for download to be finished                                                                       |
+
+The `emails.json` file should be located in the same directory where `package.json` file is located in your project and should have the following structure:
```

Comment:
line is not limited to 80 characters. we could also mention that the emails file can be modified mid-run if the user wants to test with dynamic incoming messages